src/account/SupportContact.tsx

```diff
@@ -1,4 +1,4 @@
-import { anonymizedPhone } from '@celo/utils/lib/phoneNumbers'
+import { anonymizedPhone } from '@celo/base/lib/phoneNumbers'
```

Comment:
Looks like this should be exported here https://github.com/celo-org/celo-monorepo/blob/contractkit-v2.3.0/packages/sdk/base/src/index.ts#L14, maybe drop lib/phoneNumbers so we don't rely on specific paths on the package?

Reply:
I can do that in a followup PR